### PR TITLE
#160886981 Create /auth/signup , /auth/login endpoints

### PR DESCRIPTION
**Pull request by sekayasin**
## What does this PR do?
Have signup and login for the fast-food-fast API

### Description of the task to be completed
POST /auth/signup
POST /auth/login
With those endpoints, a user can successfully create an account on fast-food-fast, the data is successfully stored in fast-food-fast db.

POST /auth/login - This endpoint at moment do tests with a username=test and password=test and generate an access token with jwt successfully.

### CODE REVIEW

**Comment by Kasulejoseph** [comment]
- And always don't forget to add a screenshot of your postman
- Add a story id as well

**Comment by sekayasin** [comment]
ooh yes, @Kasulejoseph thanks

File: api/routes.py [hunk 1]
```diff
@@ -24,73 +33,77 @@ def index():
...
+    if not password:
+        return jsonify({"msg": "Missing password parameter"}), status.HTTP_400_BAD_REQUEST
+    if username != 'test' or password != 'test':
+        return jsonify({"msg": "Bad username or password"}), status.HTTP_401_UNAUTHORIZED
+
+    # Identity can be any data that is json serializable
+    access_token = create_access_token(identity=username)
+    return jsonify(access_token=access_token), status.HTTP_200_OK
+
+# Protect a view with jwt_required, which requires a valid access token
```
Comment (Kasulejoseph) [inline api/routes.py]: In-line comments is also not recommended in PEP8 so avoid it

Comment (sekayasin) [reply api/routes.py]: Yes

**Pull request merged**